### ISSUE TITLE
OO

### ISSUE THREAD

**Issue by annefou** [NordicESMhub/deep_python#6]
Python object:
- introduction to OO with Python 
- How and why writing classes

**Comment by yellowchocobo** [comment]
I have to admit that I have been pretty bad at using classes in Python. Could you cover that in the more advanced part of the lecture?

**Comment by annefou** [comment]
Yes, I'll cover it in the second part of the course. I just added it here because I would like to have one single python lesson for CEED. I am hoping it can become a working "document" to help CEED with python dvpt.

**Comment by annefou** [comment]
The current OO episode needs to be updated as there are bugs:
- inconsistent use of object
- examples do not run properly
